### DISCUSSION TITLE
Add task list information to the project

### DISCUSSION THREAD

**Discussion by azenMatt** [community/community#4257] [Projects and Issues]
I've got a project that lists all the big epics or initiatives for my team and really love seeing the work broken down by the latest additions to task lists.

It would be awesome if both the progress of an issue's tasks, as well as the tracked / parent issue, was available to add as a column in the table view or as additional card data on the board view.

**Comment by Seraf** [comment]
Hello,

This would be a really nice addition. We are using a lot of Epics issues as we are currently using ZenHub.
We would like to mimic their Epic feature but using the Tasks list instead.

The way they are handling progress bars is based on story estimation points. So each time someone closes a sub-issue related to the epic, It makes the epic progressing with the right percentage according to the story point the issue had compared to the addition of all the story points of issues contained in the epic.

![epic-points-estimated](https://user-images.githubusercontent.com/393324/132314472-8498061f-cd32-4907-afff-a46c36488019.png)


I think that's even more precise than just having the progress bar on the task list, but it complicates a bit more the implementation ... :sweat_smile:

**Comment by 0Grit** [comment]
Being able to see this in the sheet view would be EXTREMELY helpful to my planning process.

https://github.com/orgs/golioth/projects/4/views/12
